roro/auth.py

```diff
@@ -64,11 +64,15 @@ def __init__(self, file=None):
         # work-around to fix that
         if file is None:
             file = self.find_default_file()
-        super().__init__(file=file)
+
+        if PY2:
+            _netrc.__init__(self, file=file)
+        else:
+            super().__init__(file=file)
 
     def find_default_file(self):
         filename = "_netrc" if sys.platform == 'win32' else ".netrc"
-        p = pathlib.Path.home().joinpath(filename)
+        p = pathlib.Path(os.path.expanduser('~')).joinpath(filename)
```

Comment:
What is this an issue?

Reply:
Python 2 implementation of Pathlib does not contain `.home()`

Reply:
Ok. your suggestion seems to be working Windows.

Then we don't really need `pathlib`. Could you please get rid of `pathlb` and do it using the `os` module?

Reply:
I tried it on Windows before implementing.

Remove `pathlib` from this module only right?